src/test/scala/edu/berkeley/ce/rockslicing/JointGeneratorSpec.scala

```diff
@@ -0,0 +1,347 @@
+package edu.berkeley.ce.rockslicing
+
+import org.scalatest.FunSuite
+import scala.math.sqrt
+
+class JointGeneratorSpec extends FunSuite {
+
+  private def compareJoints(joint1: Joint, joint2: Joint): Boolean = {
```

Comment:
Name this something different to indicate that it's an equality check and not a comparison.